src/plugins/kibana_react/public/util/test_helpers/react_mount_serializer.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
You should be able to get around the ESLint override by making these individual exports:

```ts
export function test(value: any) {
  return value && value.__reactMount__;
}

export function print(value: any, serialize: any, indent: any) {
  return `reactMount(${serialize(value.__reactMount__)})`;
}
```

Reply:
The way serializer works seems to require a `default` export ( they are `import serializer from XXX`, not `import * as serializer from XXX`) So I think this would not work?

Reply:
Ok nevermind, just tested it, does work. thanks.